### PR TITLE
ls: add Chinese translation

### PR DESCRIPTION
**Pull request by dengshenkk**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Review by einverne — CHANGES_REQUESTED** [review]
Thank you for the new translation, however `ls` command already in `pages.zh/common` folder.

**Comment by marchersimon** [comment]
@dengshenkk could you please apply the suggestions above?

File: pages.zh/linux/ls.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- 列出所有文件, 并在文件类型后面添加对应类型的符号(如目录会加上"/"):
+
+`ls -F`
+
+- 以人类可读的文件大小单位列出文件列表(KB,MB,GB等)
+
+`ls -lh`
+
+- 列出所有文件, 并按文件大小降序排序:
```
Comment (navarroaxel) [inline pages.zh/linux/ls.md:26]:
```suggestion
- 列出所有文件, 并按文件大小降序排序：
```

File: pages.zh/linux/ls.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`ls -lS`
+
+- 列出所有文件, 并按修改时间降序排序:
```
Comment (navarroaxel) [inline pages.zh/linux/ls.md:30]:
```suggestion
- 列出所有文件, 并按修改时间降序排序：
```

File: pages.zh/linux/ls.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- 列出目录中的文件, 每一行列出一个:
+
+`ls -1`
+
+- 列出所有文件, 包含隐藏文件:
+
+`ls -a`
+
+- 列出所有文件, 包含隐藏文件以及显示权限,拥有者,文件大小和修改日期:
```
Comment (navarroaxel) [inline pages.zh/linux/ls.md:14]:
```suggestion
- 列出所有文件, 包含隐藏文件以及显示权限,拥有者,文件大小和修改日期：
```

File: pages.zh/linux/ls.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# ls
+
+> 列出当前路径下的文件/文件夹信息。
+> 了解更多：<https://www.gnu.org/software/coreutils/ls>.
+
+- 列出目录中的文件, 每一行列出一个:
+
+`ls -1`
+
+- 列出所有文件, 包含隐藏文件:
```
Comment (navarroaxel) [inline pages.zh/linux/ls.md:10]:
```suggestion
- 列出所有文件, 包含隐藏文件：
```

File: pages.zh/linux/ls.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- 列出所有文件, 包含隐藏文件以及显示权限,拥有者,文件大小和修改日期:
+
+`ls -la`
+
+- 列出所有文件, 并在文件类型后面添加对应类型的符号(如目录会加上"/"):
+
+`ls -F`
+
+- 以人类可读的文件大小单位列出文件列表(KB,MB,GB等)
```
Comment (navarroaxel) [inline pages.zh/linux/ls.md:22]:
```suggestion
- 以人类可读的文件大小单位列出文件列表(KB,MB,GB等)：
```

File: pages.zh/linux/ls.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- 列出所有文件, 包含隐藏文件:
+
+`ls -a`
+
+- 列出所有文件, 包含隐藏文件以及显示权限,拥有者,文件大小和修改日期:
+
+`ls -la`
+
+- 列出所有文件, 并在文件类型后面添加对应类型的符号(如目录会加上"/"):
```
Comment (navarroaxel) [inline pages.zh/linux/ls.md:18]:
```suggestion
- 列出所有文件, 并在文件类型后面添加对应类型的符号(如目录会加上"/")：
```

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — CHANGES_REQUESTED** [review]
Wait. There's a few things wrong with this.
First, there is no `ls` in the `linux` directory. Second, some examples are in the wrong order, and the last one is missing completely

**Comment by marchersimon** [comment]
@dengshenkk could you please make sure your page is an exact translation of [this page](https://github.com/tldr-pages/tldr/blob/main/pages/common/ls.md)? Otherwise we would have to close this PR.

**Comment by marchersimon** [comment]
Ping @dengshenkk

**Comment by CleanMachine1** [comment]
I think the waiting tag should be added, I think it is entirely the purpose of the stalebot

**Comment by dengshenkk** [comment]
@marchersimon I don't how do it about this https://github.com/tldr-pages/tldr/pull/8143#issuecomment-1162817211

**Pull request closed without merging**